components/ResultsTable.go

```diff
@@ -1220,7 +1220,7 @@ func (table *ResultsTable) search() {
 
 	table.Filter.Input.SetAutocompleteFunc(func(currentText string) []string {
 		split := strings.Split(currentText, " ")
-		comparators := []string{"=", "!=", ">", "<", ">=", "<=", "LIKE", "NOT LIKE", "IN", "NOT IN", "IS", "IS NOT", "BETWEEN", "NOT BETWEEN"}
+		comparators := []string{"=", "!=", ">", "<", ">=", "<=", "LIKE", "ILIKE", "NOT LIKE", "IN", "NOT IN", "IS", "IS NOT", "BETWEEN", "NOT BETWEEN"}
```

Comment:
I'm surprised do not see REGEXP and NOT REGEXP

Reply:
Added them as well while I was at it...